### DISCUSSION TITLE
Notifications?

### DISCUSSION THREAD

**Discussion by raae** [raae/nattermob.dev#10] [Ideas]
The YouTube notifications are notoriously bad, we could build some form of notifications.

Email, sms, browser notifications? 

What would you like, dear viewer?

**Comment by PaulieScanlon** [comment]
Just an idea but maybe we make Nattermob a PWA and then enable push notifications. it'll only notify on mobile but might be kinda cool?

**Reply by raae** [reply]
Yes, that would be super cool and something I would like to know how to do for other projects as well!
